app/controllers/pages_controller.rb

```diff
@@ -12,7 +12,7 @@ def home
 
    def profile
     @user = current_user
-    start_date = params.fetch(:start_date, Date.today).to_date
-    @bookings = Booking.where(user: @user, start_time: start_date.beginning_of_month.beginning_of_week..start_date.end_of_month.end_of_week)
```

Comment:
I think I had line 16 because it was necessary for either flatpickr calendar on booking form or simple_calendar on profile. I'm not sure now which one is using it but we need this line I think!

Reply:
ok, put it back